### PR TITLE
Add scary story "Prison escape"

### PR DESCRIPTION
**Pull request by Mosexelle**
New scary story "Prison escape".
This is not my story, if I'm not mistaken it's Alfred Hitchcock. It's pretty short and scary, but it's a good fit for a storyteller like Monika.

### CODE REVIEW

File: Monika After Story/game/script-stories.rpy [hunk 1]
```diff
@@ -1809,3 +1809,53 @@ label mas_scary_story_flowered_lantern_3:
...
+    m 3esc "She got up, picked the lock of her cell, and slowly walked down the hallway."
+    m 3wsd "She was nearly caught a couple of times. Her heart was beating fast."
+    m 3esc "She opened the door to the darkened room where the coffins were kept."
+    m "Quietly in the dark, she found the coffin that contained the dead body"
+    extend m 3euc ", carefully climbed into it and pulled the lid shut to wait for the caretaker to come and nail the lid down."
+    m 4eua "Soon she heard footsteps and the pounding of the hammer and nails."
+    m 4esc "Even though she was very uncomfortable in the coffin with the dead body beneath her, she knew that with each nail she was one step closer to freedom."
+    m 1esc "The coffin was lifted onto the wagon and taken outside to the graveyard. She could feel the coffin being lowered into the ground."
+    m 2ekd "She didn't make a sound as the coffin hit the bottom of the grave with a thud."
+    m "Finally she heard the dirt dropping onto the top of the wooden coffin"
```
Comment (my-otter-self) [inline Monika After Story/game/script-stories.rpy:1854]: `    m "Finally she heard the dirt dropping onto the top of the wooden coffin{w=0.1}{nw}"`

File: Monika After Story/game/script-stories.rpy [hunk 2]
```diff
@@ -1809,3 +1809,53 @@ label mas_scary_story_flowered_lantern_3:
...
+    m "He would then bury the coffin."
+    m 3esc "The woman knew there would be enough air for her to breathe until later in the evening when the caretaker would return to the graveyard under cover of darkness, dig up the coffin, and set her free."
+    m 2gsc "The caretaker was reluctant to go along with this plan"
+    extend m 4tsc ", but since he and the woman had become good friends over the years, he agreed to do it."
+    m 1tsc "The woman waited several months for one of the other prison inmates to die."
+    m 1esc "One night, she was asleep in her cell when she heard the death bell ringing."
+    m 3esc "She got up, picked the lock of her cell, and slowly walked down the hallway."
+    m 3wsd "She was nearly caught a couple of times. Her heart was beating fast."
+    m 3esc "She opened the door to the darkened room where the coffins were kept."
+    m "Quietly in the dark, she found the coffin that contained the dead body"
```
Comment (my-otter-self) [inline Monika After Story/game/script-stories.rpy:1848]: `    m "Quietly in the dark, she found the coffin that contained the dead body{w=0.1}{nw}"`

**Review by my-otter-self — COMMENTED** [review]
2 extend fixes, adding pauses and nws.

File: Monika After Story/game/script-stories.rpy [hunk 3]
```diff
@@ -1869,3 +1869,50 @@ label mas_scary_story_flowered_lantern_3:
...
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_story_database,
+            eventlabel="mas_scary_story_prison_escape",
+            category=[store.mas_stories.TYPE_SCARY],
+            prompt="Prison Escape",
+            unlocked=False
+        ),
+    code="STY"
```
Comment (multimokia) [inline Monika After Story/game/script-stories.rpy:1882]:
```suggestion
        code="STY"
```

**Review by multimokia — CHANGES_REQUESTED** [review]
smol indent mismatch

**Review by jmwall24 — APPROVED** [review]

**Review by multimokia — APPROVED** [review]

**Pull request merged**